INSTALL.md

```diff
@@ -67,6 +67,17 @@ To install Postgresql, follow instructions from the [Postgresql download page](h
 - `clang-format-12` (for `make format` to work)
 - `sed` and `perl`
 - `libunwind-dev`
+- Rust toolchain (see [Installing Rust](#installing-rust) subsection)
+  - `cargo` >= 1.63
+  - `rust` >= 1.63
```

Comment:
I think its 1.74 version now, https://github.com/stellar/stellar-core/blob/master/rust-toolchain.toml#L2

Reply:
Got it! Fixed in 79630f67d59ba21216ec4f404a3d145b8232a0ab